pyscf/dft/libxc.py

```diff
@@ -770,7 +770,7 @@ def _xc_key_without_underscore(xc_keys):
 # Issue 1480
 if not hasattr(__config__, 'B3LYP_WITH_VWN5'):
     warnings.warn('Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, '
-                  'the same to the B3LYP functional in Gaussian and ORCA (issue 1480). '
+                  'the same to the B3LYP functional in Gaussian (issue 1480). '
```

Comment:
this should read: the original B3LYP functional as published by Stephens et al.

Reply:
After some struggling, I can understand that your 'the B3LYP functional' of ORCA refers to the one called 'B3LYP_G' in ORCA manual, does it? That makes sense, but it will be very hard to understand as a warning message.

Reply:
?

Reply:
I mean, the B3LYP keyword (a wrong B3LYP definition) of ORCA refers to B3LYP5 (although it has a right B3LYP definition called B3LYP_G), so it's misleading to state here that current PySCF's B3LYP is the same to the one in ORCA.

Reply:
Yes, ORCA, TURBOMOLE etc don't implement B3LYP but B3LYP5 which is a different functional. They are wrong.

Reply:
So could this conversation being resolved? Removing 'and ORCA' here is better, right?

Reply:
It is better, but "same as B3LYP functional in Gaussian" is still not saying the main point, which is that that implementation is the correct one.

Reply:
What about "corresponding to the original definition by Stephens et al. (issue 1480) and the same as the B3LYP functional in Gaussian"

Reply:
@hebrewsnabla yes!